core/ui/PageControls/layout.tid

```diff
@@ -0,0 +1,23 @@
+title: $:/core/ui/Buttons/layout
+tags: $:/tags/PageControls
+caption: {{$:/core/images/layout-button}} {{$:/language/Buttons/LayoutSwitcher/Caption}}
+description: {{$:/language/LayoutSwitcher/Description}}
+list-after: $:/core/ui/Buttons/theme
```

Comment:
We shouldn't have a list-after field here. The correct way to set the default ordering is via the tiddler https://github.com/Jermolene/TiddlyWiki5/blob/master/core/wiki/tags/PageControls.tid

Reply:
Ok, I should check if its tag has a list field in the future.